maps/cam_lao_nam/mission.sqm

```diff
@@ -4969,7 +4969,7 @@ class Mission
 										class data
 										{
 											singleType="STRING";
-											value="custom\billboards\gh_bb.paa";
+											value="custom\billboards\supporter_sign.paa";
```

Comment:
paa path here doesn't match the paa path of added file -- `mission/custom/billboards/Supporter_SIgn.paa`

is that an issue or does it work fine? (i'll find out tomorrow anyway).

Reply:
it worked on my end.

Reply:
yeah looks fine